pyoverkiz/client.py

```diff
@@ -298,7 +299,7 @@ async def cozytouch_login(self) -> str:
 
             jwt = jwt.strip('"')  # Remove surrounding quotes
 
-            return jwt
+            return str(jwt)
```

Comment:
mypy: pyoverkiz/client.py:302: error: Returning Any from function declared to return "str"  [no-any-return]

Reply:
We should cast it if we know it is a string for sure.